### DISCUSSION TITLE
Number of active Logs AnomalyDetectors exceeds allowed limit

### DISCUSSION THREAD

**Discussion by petro2050** [aws/aws-cdk#29579] [Q&A]
I'm getting this error when deploying CDK. What is the limit on number of active Logs AnomalyDetectors? Also, is it possible to increase this limit?

**Comment by mpremo** [comment]
Same issue also when creating via CloudFormation.

"Resource handler returned message: "Number of active Logs AnomalyDetectors exceeds allowed limit."

When I create log groups manually and assign anomaly detectors to them, I can go past this limit.  So maybe it's an undocumented CDK/CF limitation?
